core/src/main/scala/fr/davit/akka/http/metrics/core/HttpMetricsNames.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 Michel Davit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.davit.akka.http.metrics.core
+
+case class HttpMetricsNames(
```

Comment:
Would be nice to define a trait and have this as hidden implementation.

I know some users extends the registry to have additional metrics. If they want to keep the structure, they must be able to extend this `HttpMetricsNames`